src/basics.php

```diff
@@ -143,6 +143,15 @@ function dd($var, $showHtml = null): void
             'file' => $trace[0]['file'],
         ];
 
+        if (PHP_SAPI !== 'cli') {
```

Comment:
You could add here

    && !empty($_SERVER['HTTP_REFERER'])

this way you don't need env() usage either.